lib/unit-ruby/util/api_resource.rb

```diff
@@ -96,7 +96,7 @@ def self.belongs_to(resource_name, class_name: nil)
 
       define_method("#{resource_name}=") do |resource|
         relationships[resource_name] = {
-          data: { type: resource_name, id: resource.id }
+          data: { type: resource_name.to_s.camelize(:lower).to_sym, id: resource.id }
```

Comment:
Could this have just been `type: resource_type`?

Reply:
It could not. For relationships there might be a different between the resource type and the relationship name. For example an `account` has a relationship to an `IndividualCustomer` resource, but the relationship is to a `customer`.

I needed to make the change above to also support having a relationship to a `white_label_theme` which translates to a type of `whiteLabelTheme`.

Perhaps I should've renamed resource_name everywhere in the method to `relationship_name`